packages/react/src/components/NumberInput/NumberInput-story.js

```diff
@@ -88,6 +89,7 @@ export const Default = () => {
       label="NumberInput label"
       helperText="Optional helper text."
       invalidText="Number is not valid"
+      onChange={(e, something) => console.log(something)} //TO-DO: remove before merging
```

Comment:
```suggestion
```

FYI storybook already shows the event object in the actions tab so this is probably not needed for testing!